### PR TITLE
INTYGFV-15700: Added error modals

### PR DESCRIPTION
**Pull request by mhornfeldt**

### CODE REVIEW

File: apps/rehabstod/src/pages/Patient/components/patientOverview/PatientOverview.tsx [hunk 1]
```diff
@@ -46,37 +47,60 @@ export function PatientOverview({
...
-        subTitle="Vårdgivare"
-        description="Det finns spärrad intygsinformation hos andra vårdgivare. Endast patienten kan häva spärren genom att kontakta den enhet där spärren sattes. Klicka nedan för att visa vilka vårdgivare som har spärrad information."
+    )
+  }
+
+  return (
+    <>
+      <ErrorModal
+        show={giveConsentError !== undefined || (consentResponse && consentResponse.responseCode !== 'OK')}
+        description="Tyvärr kan samtycke inte registreras på grund av ett tekniskt fel. Försök igen om en stund."
```
Comment (ChrilleSigerhed) [inline apps/rehabstod/src/pages/Patient/components/patientOverview/PatientOverview.tsx:69]: Is this text correct? According to the "krav" it should be

```Samtycke kan inte registreras på grund av ett tekniskt fel. Försök igen om en stund. Om problemet kvarstår, kontakta i första hand din lokala IT-support och i andra hand <Inera Support>.```

File: apps/rehabstod/src/pages/Patient/components/patientOverview/PatientOverview.tsx [hunk 2]
```diff
@@ -46,37 +47,60 @@ export function PatientOverview({
...
-        subTitle="Vårdenhet"
-        description="Det finns spärrad information hos en annan vårdenhet inom din vårdgivare. Endast patienten kan få spärren hävd genom att kontakta den enhet där spärren sattes. Du kan klicka nedan för att visa vilka vårdenheter som har spärrad information hos sig."
-        items={sjfMetaData.vardenheterInomVGMedSparr}
+  if (sjfMetaData.consentServiceError || sjfMetaData.blockingServiceError) {
+    return (
+      <ErrorModal
+        show={sjfMetaData.consentServiceError || sjfMetaData.blockingServiceError}
+        description={
+          sjfMetaData.consentServiceError
+            ? 'Tyvärr kan information från andra vårdgivare inte hämtas på grund av ett tekniskt fel. Försök igen om en stund.'
```
Comment (ChrilleSigerhed) [inline apps/rehabstod/src/pages/Patient/components/patientOverview/PatientOverview.tsx:56]: Is this text correct? According to the "krav" it should be

```Information från andra vårdgivare kan inte hämtas på grund av ett tekniskt fel. Försök igen om en stund. Om problemet kvarstår, kontakta i första hand din lokala IT-support och i andra hand <Inera Support>.```

**Review by ChrilleSigerhed — CHANGES_REQUESTED** [review]
Awsome, really great work! I liked that u created a separate component for ErrorId, this should be incorperated in the related error-handling tasks we have implemented aswell!

I really like the implementation and the look and feel of the modal, works like charm!

I added some comments regarding the texts, If the case is that the "krav" has changed, I suggest we update them asap!

Regarding the modal for this error: ```Något har blivit fel med anrop till Spärrtjänsten. Felet ligger hos dem, inte oss.```. I cant find this text, am I missing it or should we just ignore it cause its not relevant?

File: apps/rehabstod/src/pages/Patient/components/patientOverview/PatientOverview.tsx (hunk 2 above)
Comment (mhornfeldt) [reply apps/rehabstod/src/pages/Patient/components/patientOverview/PatientOverview.tsx:56]: Looks like it has updated while the PR has been open, I'll fix the texts

Comment (mhornfeldt) [reply apps/rehabstod/src/pages/Patient/components/patientOverview/PatientOverview.tsx]: "Något har blivit fel med anrop till Spärrtjänsten. Felet ligger hos dem, inte oss." is when blockingServiceError is true

File: apps/rehabstod/src/error/ErrorId.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  }
+
+  return (
+    <>
+      <div className="mt-5 flex">
+        <p className="mr-2 font-bold">FEL-ID:</p>
+        {errorId}
+        <TooltipIcon
+          description="Kopiera fel-id"
+          name="copy-link"
```
Comment (ChrilleSigerhed) [inline apps/rehabstod/src/error/ErrorId.tsx:19]: After speaking with UX, we have decided that the tooltip icon should be updated to ```name="copy-file"```

**Review by ChrilleSigerhed — CHANGES_REQUESTED** [review]

**Pull request merged**